test/Data/Functor/Listable.hs

```diff
@@ -230,12 +230,6 @@ instance Listable Language.Language where
     \/ cons0 Language.Ruby
     \/ cons0 Language.TypeScript
 
-instance Listable (f a) => Listable (App f a) where
```

Comment:
Does `App` even still exist? `Ap` is in `base` now, so if it does, maybe it shouldn’t.

Reply:
`App` does exist. I think it is fundamentally different from `Data.Monoid.Ap` though. `App` uses `*>`, whereas `Ap` uses `liftA2 (<>)`.

Reply:
Ahh indeed, it’s `AppMerge` that’s the same as `Ap`, my bad.